tests/unittests/graphSchedulerTest.cpp

```diff
@@ -87,3 +89,47 @@ TEST(GraphScheduler, testMaxSizeLessThanResultSize) {
   EXPECT_LT(std::distance(schedule.begin(), sliceBigIt),
             std::distance(schedule.begin(), concatSmallIt));
 }
+
+TEST(GraphScheduler, testTopologicalSort) {
+  Module MD;
+  Context ctx;
+  auto *smallTensorA =
+      MD.createPlaceholder(ElemKind::FloatTy, {1, 4, 4}, "small_1", false);
+  ctx.allocate(smallTensorA);
```

Comment:
You don't need to allocate anything into the context here because you are not executing anything.

Reply:
Done